service/k8sclient/kubectl/kubectl.go

```diff
@@ -122,24 +127,30 @@ func getKubectlCmd(ctx context.Context) ([]string, error) {
 		return nil, err
 	}
 
-	return lookupCorrectKubectlCmd(kubectlCmdNames)
+	return lookupCorrectKubectlCmd(append(kubectlCmdNames, defaultKubectl, minikubeDevKubectl))
 }
 
 func lookupCorrectKubectlCmd(kubectlCmdNames []string) ([]string, error) {
 	for _, kubectlCmdName := range kubectlCmdNames {
-		kubectlPath, err := exec.LookPath(kubectlCmdName)
+		cmd := strings.Split(kubectlCmdName, " ")
+		kubectlPath, err := exec.LookPath(cmd[0])
```

Comment:
LookPath will not work because `/opt/dbaas-tools/bin` is not in path, and should not be.

We also should add `/opt/dbaas-tools/bin` to cmd'e environment in `run` to handle iam-aws-authenticator and other tools

Reply:
> If file contains a slash, it is tried directly and the PATH is not consulted. The result may be an absolute path or a path relative to the current directory.

I pass absolute path to the binary.

Reply:
Yes, but we can't expect kubeconfig file to use `/opt/dbaas-tools/bin/iam-aws-authenticator` instead of just `iam-aws-authenticator`. So we should run `/opt/dbaas-tools/bin/kubectl-XXX` but with modified environment so `iam-aws-authenticator` can be found

Reply:
My answer was for the first, I'm totally agree with a second part related to iam-aws-authenticator.

Reply:
So we use LookPath only to check that it exist?

Reply:
Ah, and to lookup `minikube`. Got it. Maybe a comment would help :)